plugins/MauticCrmBundle/Integration/CrmAbstractIntegration.php

```diff
@@ -432,6 +432,10 @@ public function getMauticLead($data, $persist = true, $socialCache = null, $iden
                 return;
             }
 
+            if (isset($fieldsToUpdateInMautic[0])) {
```

Comment:
@kuzmany we are 100% sure there is no problem with other integration ? Maybe we should check if we are in hubspot context ?